test/system/deleted-files.spec.ts

```diff
@@ -0,0 +1,66 @@
+import * as path from "path";
+import * as plugin from "../../lib";
+
+describe("scan result should not contain any deleted file", () => {
```

Comment:
The test description from `describe` and then `it` test below in line 15 should form a sentence describing a test case. Now we have: 
`describe("scan result should not contain any deleted file"` `it("scan result don't bring deleted facts"`
and ideally we would want to have something more like: 
`describe("deleted files"` `it("should not exist in scan result"` - so the `describe` tells us what the whole suite does and then we can keep adding more tests for the deleting logic. 
Hope that makes sense? :-)